cmd/manager/main.go

```diff
@@ -47,52 +57,74 @@ func printVersion() {
 	logrus.Printf("operator-sdk Version: %v", sdkVersion.Version)
 }
 
+func setupScheme() *kruntime.Scheme {
+	scheme := kruntime.NewScheme()
+
+	utilruntime.Must(apis.AddToScheme(scheme))
+	utilruntime.Must(olmv1alpha1.AddToScheme(scheme))
+	utilruntime.Must(v1beta1.AddToScheme(scheme))
+	utilruntime.Must(corev1.AddToScheme(scheme))
+
+	if configv1.IsAPIAvailable() {
```

Comment:
Are the `configv1` schemes actually **required** for the controller to run, or are they just optional?

Reply:
I'm not entirely sure tbh - for the operatorhub controller they're absolutely necessary as the OperatorHub Go types are housed in [o/api](https://github.com/openshift/api/blob/master/config/v1/types_operatorhub.go). I'd like to hold off on modifying this behavior as much as possible as we'll need to backport these fixes.